tests/TestCase/View/Helper/PaginatorHelperTest.php

```diff
@@ -2833,7 +2833,7 @@ public function testHasPage()
         $result = $this->Paginator->hasPage('Article', 2);
         $this->assertTrue($result);
 
-        $result = $this->Paginator->hasPage(2);
+        $result = $this->Paginator->hasPage(null, 2);
```

Comment:
This change makes the standard use case (single model pagination) more clunky. This feels like a mixed type we should keep.

Reply:
But it is completely undocumented so far.

And this now in sync with all the other methods that have the model as first key:
- hasPrev($model = null)
- hasNext($model = null)
- sortKey($model = null
- sortDir($model = null

If anything, the order should be inverse: `$page, $model = null` That would make way more sense :)
Similar to `public function param($key, $model = null)`

Reply:
> Can we switch the order with some migration guide?

If we were going to make a change now is the time to do so.